test/mocks/mock_server.ex

```diff
@@ -0,0 +1,18 @@
+defmodule MockServer do
```

Comment:
why do we need this?

Reply:
It is just a "dummy" GenServer to represent any persistant process. It can be replaced with an agent or any other spawned process that doesn't terminate. A GenServer was chosen to represent a Station or a Query Collector.